### ISSUE TITLE
[Feature request] add notification

### ISSUE THREAD

**Issue by Sukii23** [tumuyan/RealSR-NCNN-Android#34]
when there is no notification in the application, android ends up closing the application in the background more easily. please add a notification when starting the upscale

**Comment by tumuyan** [comment]
good idea. but can a foreground app really work well when this App is running normally in the background?

**Comment by Sukii23** [comment]
there are some slight crashes if the application runs in the background while using another in the foreground. but this option could not be optional to activate? from android 10 if there is a notification android does not close the service, even if it keeps the device in general slower.. this is useful for when you want to make sure that it will not be closed or interrupted in some way while it is running.

**Comment by tumuyan** [comment]
I have add notification but not adjust the structure. maybe it useless for keep alive for foreground scene.

**Comment by Sukii23** [comment]
I'm going to do some tests and see if the service remains in the background with the notification

**Comment by Sukii23** [comment]
Now the application is no longer open, not even in the foreground. Before, when I was going to convert, I left the cell phone off with the application in the foreground and when I turned it on, everything worked normally. Now for some reason it closes the service both in the foreground (screen off) and in the background. detail: 4 ram

**Comment by KSEZU** [comment]
Update for this? As I am writing this comment the app will stop running but when I go back to the app again it will continue, we really need the app to run in background
